src/realm/util/ez_websocket.cpp

```diff
@@ -78,6 +85,19 @@ class EZSocketImpl final : public EZSocket, public websocket::Config {
         return m_observer.websocket_binary_message_received(ptr, size);
     }
 
+    /// Return the location field in the response headers
+    std::string_view websocket_get_location_header(const util::HTTPHeaders* headers)
+    {
+        if (headers != nullptr) {
+            // HTTPHeaders does a case-insensitive search
+            auto location = headers->find("location");
+            if (location != headers->end()) {
+                return location->second;
+            }
+        }
+        return "";
```

Comment:
`return {}` or `return nullptr`. and can we swap the order of checking so it's `if (!headers) { return {}; } ... rest of function`?

Reply:
Agree - will update